example_deploy.sh

```diff
@@ -11,68 +11,73 @@ kubectl apply -f - <<EOF
 apiVersion: apps/v1
 kind: Deployment
 metadata:
-  name: ${APP}
+  name: ${APP}-without-attach
   namespace: ${NAMESPACE}
   labels:
-    app: ${APP}
+    app: ${APP}-without-attach
+    service: ${APP}-without-attach
   annotations:
     deployment.kubernetes.io/revision: "$revision"
 spec:
   replicas: 1
   selector:
     matchLabels:
-      app: ${APP}
+      app: ${APP}-without-attach
   template:
     metadata:
-      name: ${APP}
       labels:
-        app: ${APP}
+        app: ${APP}-without-attach
+        service: ${APP}-without-attach
       annotations:
         deployment.kubernetes.io/revision: "$revision"
     spec:
+      dnsPolicy: ClusterFirstWithHostNet
       containers:
-        - name: example-app
-          image: hashicorp/http-echo:alpine
-          imagePullPolicy: Always
-          args:
-          - "-text='hello world'"
-          ports:
-          - containerPort: 5678
+      - name: petclinic
```

Comment:
Just missing this one 🙂 
```suggestion
      - name: ${APP}
```